design-tokens/tokens/layer1/base.json

```diff
@@ -1,109 +1,129 @@
 {
-  "size": {
-    "dimension": {
-      "scale": {
-        "value": "2",
+  "dimension": {
+    "space": {
+      "0": {
+        "value": "0px",
         "type": "dimension"
       },
-      "xs": {
-        "value": "4",
+      "25": {
+        "value": "0.125rem",
         "type": "dimension"
       },
-      "sm": {
-        "value": "{size.dimension.xs} * {size.dimension.scale}",
+      "50": {
+        "value": "0.25rem",
         "type": "dimension"
       },
-      "md": {
-        "value": "{size.dimension.sm} * {size.dimension.scale}",
+      "75": {
+        "value": "0.375rem",
         "type": "dimension"
       },
-      "lg": {
-        "value": "{size.dimension.md} * {size.dimension.scale}",
+      "100": {
+        "value": "0.5rem",
         "type": "dimension"
       },
-      "xl": {
-        "value": "{size.dimension.lg} * {size.dimension.scale}",
+      "125": {
+        "value": "0.75rem",
         "type": "dimension"
-      }
-    },
-    "spacing": {
-      "xs": {
-        "value": "{size.dimension.xs}",
-        "type": "spacing"
       },
-      "sm": {
-        "value": "{size.dimension.sm}",
-        "type": "spacing"
+      "150": {
+        "value": "1.25rem",
+        "type": "dimension"
       },
```

Comment:
This one should be "250" to respect the percentage scale you told me about

Reply:
Thanks, Updated